VLC-iOS-Bridging-Header.h

```diff
@@ -9,6 +9,7 @@
 #import "VLCConstants.h"
 #import "VLCDownloadViewController.h"
 #import "VLCLibrarySearchDisplayDataSource.h"
+#import "VLCLibraryViewController.h"
```

Comment:
Why do you need the LibraryViewController?

Reply:
`VLCLibraryViewController.h` is where `EmptyLibraryView` is declared and used within `VLCLibraryViewController.h`. Maybe writing a new version of it in Swift is beneficial in the long run?

Reply:
Yes I think it should be taken out since `VLCLibraryViewController` will be removed "soon".

Reply:
It shouldn't have been declared in the LibraryViewController in the first place. It needs to live in it's own file. You don't have to write a new version in Swift, just move it into it's own class via copy paste :)